docs/driver.md

```diff
@@ -12,14 +12,20 @@ The command used to launch the IDE. By default, the driver uses the `idea` comma
 By default, the driver waits `30 seconds` for the IDE to connect the probe. After exceeding this time, the whole test fails.  
 #### Automatic checks
 ```
-driver.checks {
-    errors = false
+driver.check {
+     errors {
+       enabled = false
+       exclude-messages = []
+       include-messages = [".*"]
+     }
     freezes = false
 }
 ```
 
 By default, the driver doesn't fail the test upon detecting any errors or freezes during the execution.
-  
+Field `exclude-messages` is an empty list by default. Putting some regular expressions into this field will cause probe to ignore IDE errors whose messages match any of the given expressions, even if `probe.driver.check.errors.enabled` is `true`.
```

Comment:
As usual problem with regexes all of the industry... is it:
1. _whose messages **fully** match any of the given expressions_ or
2. _whose messages **contain a substring** matching any of the given expressions_ or
(I.e. _anchored_ vs _unanchored_ regex)

Pls make it clear from the docs... for this purpose, I think 2. is more natural (so that `.*` at the start/end can be avoided)

Reply:
Especially that you rather won't be fully matching a whole stacktrace :p